server/lib/src/constants/mod.rs

```diff
@@ -45,13 +45,30 @@ pub const SYSTEM_INDEX_VERSION: i64 = 31;
  */
 pub type DomainVersion = u32;
 
+/// Domain level 0 - this indicates that this instance
+/// is a new install and has never had a domain level
+/// previously.
 pub const DOMAIN_LEVEL_0: DomainVersion = 0;
```

Comment:
these are starting to feel like... numbers, or an enum is required 😄

Reply:
The reason I want them as const, is so when we *delete* them from the constants, we find all the places we were relying on the value in the code base.

Reply:
Yeah, enum would make sense in that case, but meh, yaks. 😄